src/errors.rs

```diff
@@ -32,3 +32,58 @@ pub(crate) enum ErrorKind {
     Generic,
     SysusersUnknownType,
 }
+
+/// Helper type for faliable functions within this library.
+pub type SdResult<T> = Result<T, SdError>;
```

Comment:
I'd rather not introduce this, I don't see a real need for it.

Reply:
removed